probeinterface/probe.py

```diff
@@ -916,7 +916,21 @@ def get_slice(self, selection):
         return sliced_probe
 
 
-def _2d_to_3d(data2d, plane):
+def data_2d_to_3d(data2d, plane):
+    """
+    add a third dimension
+    
+    Parameters
+    ----------
+    data2d: np.array
+        shape (n, 2)
+    plane: str
+        'xy', 'yz' or 'xz'
+    Returns
+    -------
+    data3d
+        shape (n, 3)
+    """
     data3d = np.zeros((data2d.shape[0], 3), dtype=data2d.dtype)
```

Comment:
Why not using [this logic](https://github.com/SpikeInterface/probeinterface/pull/89/files#diff-da734353bfc94648f940ed953aedd7e7db15b10086aa568a90e00a7c616493afR963) here?

Reply:
Yep it is true.